css/main.css

```diff
@@ -0,0 +1,964 @@
+}
+
+.th-title {
+  background-color: #7e3319;
+  font-size: 1.75rem;
+}
+
+.td-btn {
+  background-color: #be4c25;
+}
+
+.btn:hover {
+  background-color: #0b9ce0;
+  color: white;
+}
+
+.f-bold {
+  font-weight: 700;
+}
+
+.f-font-paragraph {
+  font-size: 1em;
+}
+
+.f-large {
+  font-size: 2em;
+}
+
+.f-small-line-height {
+  line-height: .5;
+}
+
+.f-price {
+  font-weight: 400;
+  font-size: .75em;
+  text-transform: uppercase;
+}
+
+.f-team {
+  font-size: 1.5em;
+  line-height: .1;
+  margin: .5em 0;
+}
+
+.f-client {
+  color: #fd6632;
+}
+
+.logo {
+  background-color: #fd6632;
+  padding: 1rem 1.5rem;
+  border-radius: 50%;
+  margin: 0 auto;
+}
+
+.logo:hover {
+  background-color: #0b9ce0;
+}
+
+.underline {
+  border-bottom: solid 2px #fd6632;
+}
+
+.btn {
+  background-color: #fd6632;
+  color: white;
+  font-size: .65em;
+  box-shadow: none;
+  border: none;
+  padding: .3em 1em;
+  margin: .3em auto;
+}
+
+.slider {
+  display: block;
+}
+
+.slider-break {
+  display: none;
+}
+
+progress {
+  width: 90%;
+  -webkit-appearance: none;
+  appearance: none;
+  height: .75em;
+}
+
+progress::-webkit-progress-bar {
+  background-color: #3a3a3a;
+}
+
+progress::-webkit-progress-value {
+  background-color: #fd6632;
+}
+
+.s-progress-paragraph {
+  text-align: left;
+  margin-left: 5%;
+  line-height: .1;
+  margin-bottom:  .1em;
+  margin-top: 2em;
+}
+
+.s-progress-question-text {
+  margin-top: 5em;
+}
+
+.contact-btn {
+  height: 3em;
+}
+
+/*=== FORM ===*/
+label {
+  display: inline-block;
+}
+
+input, textarea {
+  height: 3em;
+  width: 80%;
+  font-size: .80rem;
+  color: #fff;
+  border: 1px solid #7e3218;
+  margin-bottom: 5%;
+  background-color: #7e3218;
+  padding: 0 .7em;
+  display: inline;
+}
+
+.s-form-message-field {
+  height: 15em;
+  padding-top: 1em;
+  color: #fff;
+}
+
+::-webkit-input-placeholder {
+  color: #fff;
+}
+
+:-moz-placeholder {
+  color: #fff;
+}
+
+::-moz-placeholder {
+  color: #fff;
+}
+
+:-ms-input-placeholder {
+  color: #fff;
+}
+
+.s-form-btn {
+  background-color: #fff;
+  color: #7e3218;
+  height: 3em;
+}
+
+.s-theme-author {
+  text-decoration: none;
+  color: #fd6632;
+}
+
+.s-theme-author:hover {
+  color: #0b9ce0;
+}
+
+/*=== BREAKPOINTS ===*/
+@media only screen and (min-width: 667px) {
+
+  h1 {
+    font-size: 2.4em;
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  article {
+    padding: 2em 10%;
+  }
+
+  .l-flex-container {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -o-flex;
+    display: -webkit-flex;
+    display: flex;
+    -moz-flex-direction: row;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -moz-flex-flow: row wrap;
+    -ms-flex-flow: row wrap;
+    flex-wrap: wrap;
+    -webkit-justify-content: center;
+    justify-content: center;
+    -webkit-align-content: center;
+    align-content: center;
+    -webkit-align-items: center;
+    align-items: center;
+  }
+
+  #portfolio {
+    padding:2em 9%;
```

Comment:
Please be consistent in how you write your rulesets -- all of the other rulesets in this document (up until this point) are written with a space: `padding: 2em 9%`.